interfaces/policy/helpers.go

```diff
@@ -38,6 +38,8 @@ func snapIDSnapd(snapID string) bool {
 		// TODO: when snapd snap is uploaded to staging, replace this with
 		// the real snap-id.
 		"todo-staging-snapd-id",
+		// the snap-id used in the unit tests
+		"snapd-snap-ididididididididididi",
```

Comment:
I asked @zyga originally not to do this, changing the test(s) if possible seems preferable

Reply:
Yeah, I just need to track down the offending test. I will do it.

Reply:
Ah, I see @mvo5  did this already! Thank you both :)